github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -1,6 +1,36 @@
 ---
 presubmits:
   nmstate/kubernetes-nmstate:
+    - name: pull-kubernetes-nmstate-unit-test
+      skip_branches:
+        - release-\d+\.\d+
+      annotations:
+        fork-per-release: "true"
+      always_run: true
+      optional: false
+      decorate: true
+      decoration_config:
+        timeout: 3h
+        grace_period: 5m
+      max_concurrency: 6
+      labels:
+        preset-dind-enabled: "true"
+        preset-docker-mirror: "true"
+        preset-shared-images: "true"
+      spec:
+        nodeSelector:
+          type: vm
+          zone: ci
+        containers:
+          - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
+            securityContext:
+              privileged: true
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "GINKGO_EXTRA_ARGS='--compilers=2' make all test/unit"
```

Comment:
Actually, could you please change this to go through a script under `automation/`? As we do it for e2e. It would give us a chance to setup everything for standalone CI run. Plus we keep all the robots under the same hood.